pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -308,15 +307,46 @@ func getPinnedLibraries(registry string) []libInfo {
 	return res
 }
 
-// getLibrariesLanguageDetection runs process language auto-detection and returns languages to inject for APM Instrumentation.
-// The languages information is available in workloadmeta-store and attached on the pod's owner.
-func (w *Webhook) getLibrariesLanguageDetection(pod *corev1.Pod) []libInfo {
-	if config.Datadog().GetBool("admission_controller.auto_instrumentation.inject_auto_detected_libraries") {
-		// Use libraries returned by language detection for APM Instrumentation
-		return w.getAutoDetectedLibraries(pod)
-	}
+type libInfoLanguageDetection struct {
+	libs             []libInfo
+	injectionEnabled bool
 
-	return nil
+	injected bool
```

Comment:
I don't see this field populated anywhere.

Maybe you wanted to rename it to `injectionEnabled` but forgot to remove it?

It is also a bit confusing to have both `injectionEnabled` and `injected`, because why would one be true while the other one is not.